exercises/bowling/bowling.exs

```diff
@@ -1,29 +1,30 @@
 defmodule Bowling do
+  @type t :: any()
+
   @doc """
     Creates a new game of bowling that can be used to store the results of
     the game
   """
-
-  @spec start() :: any
+  @spec start() :: t()
   def start do
   end
 
   @doc """
     Records the number of pins knocked down on a single roll. Returns `any`
     unless there is something wrong with the given number of pins, in which
-    case it returns a helpful message.
+    case it returns a helpful error tuple.
   """
 
-  @spec roll(any, integer) :: any | String.t()
+  @spec roll(t(), integer()) :: t() | {:error, String.t()}
```

Comment:
Typically if there's an `{:error, _}` response, the other possible response to a function is either `:ok` or `{:ok, _}`. If we want to recommend that these functions returning tagged tuples that's fine, but we should be consistent in that case.